packages/components/src/inputs/src/InputGroup.module.css

```diff
@@ -0,0 +1,92 @@
+.hop-InputGroup  {
+    --input-group-background: var(--hop-neutral-surface);
+    --input-group-border: var(--hop-neutral-border);
+
+    position: relative;
+
+    display: inline-flex;
+    gap: var(--hop-space-inline-sm);
+    align-items: center;
+
+    box-sizing: border-box;
+    inline-size: fit-content;
+    block-size: 2.5rem;
+    padding-inline: var(--hop-space-inset-md) var(--hop-space-inset-md);
+
+    background: var(--input-group-background);
+    border: 0.0625rem solid var(--input-group-border);
+    border-radius: var(--hop-shape-rounded-md)
+}
+
+.hop-InputGroup--fluid,
+.hop-InputGroup--fluid .hop-InputGroup__input {
+    inline-size: 100%
+}
+
+.hop-InputGroup:has(.hop-InputGroup__input[data-focus-within]),
+.hop-InputGroup:has(.hop-InputGroup__input[data-hovered]) {
+    --input-group-background: var(--hop-neutral-surface-hover);
+    --input-group-border: var(--hop-neutral-border-hover)
```

Comment:
these should have the internal variable name and the values should be component variables